Documentation/intro.rst

```diff
@@ -0,0 +1,78 @@
+`Pcn-k8s` is a brand new network plug-in for *Kubernetes*, which can handle the network of an entire datacenter and that outperforms existing solutions in terms of throughput.
+`Pcn-iptables` is a more efficient and scalable clone of *iptables*.
+
+A brief overview of the Polycube layered structure, including the command line interface (CLI), standalone applications, and some of the available cubes, is shown in the picture below.
+
+.. image:: images/polycube-archi.png
+   :alt: Polycube architecture brief
+
+
+Main features
+-------------
+
+Extremely fast
+~~~~~~~~~~~~~~
+Polycube enables extremely fast and efficient network services, thanks to its capability to run inside the Linux kernel and, whenever possible, as close as possible to the network interface card driver, which reduces the time spent in ancillary components. 
+
+
+Born for service chaining
+~~~~~~~~~~~~~~~~~~~~~~~~~
+Polycube supports the definition of multiple network scenarios through the composition of many elementary building blocks (i.e., *cubes*),  which can be combined (e.g., attached one to the other) to create complex network services.
+For instance, several dockers can communicate through a bridge, which is then attached to a router to provide internet connectivity (possibly through a nat), while a firewall protects the entire infrastructure.
+
+Polycube has been designed to simplify service chaining: cubes can be dynamically instantiated and seamlessly connected together using virtual links, mimicking traditional networks in which dedicated middlebox are connected with each other through physical wires.
+As a consequence, cubes can be composed to build arbitrary service chains and provide custom network connectivity to namespaces, docker, virtual machines, and physical hosts.
+
+
+Production-grade network services
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Polycube greatly simplifies the way data, control, and management planes communicate, hence enabling the creation of rich network services that include all the above features.
+In turn, this offers a simplified environment to service developers, who can leverage the power of Polycube to write the data/control and management parts of their network services.
+
+In detail, most network services include a *data plane*, such as the longest prefix match algorithm in a router, a *control plane*, e.g., where routing protocols are executed, and a *management plane*, devoted to the configuration and monitoring of the service.
+`bpf` aims at the creation of fast data planes, leaving the rest under the responsibility of the developer; Polycube overcomes this limitation with a rich set of primitives natively provided by the framework.
+In addition, Polycube provides the software infrastructure required to overcome possible limitations of BPF in the data plane (e.g., the limited size of data plane programs), enabling the steering of packets that require complex processing in user-space, where previous limitations doesn't apply.
```

Comment:
limitations `doesn't` -> `don't` apply

Reply:
It should be fine, third-person speech used here.